reference/tests/snapshots/snapshot__run_reference_examples@introduction-0.prql.snap

```diff
@@ -30,10 +30,17 @@ WITH table_0 AS (
     20
 )
 SELECT
-  table_0.*,
+  title,
+  AVG(salary),
+  AVG(salary + payroll_tax),
+  SUM(salary),
+  SUM(salary + payroll_tax),
+  AVG(salary + payroll_tax + benefits_cost),
+  sum_gross_cost,
```

Comment:
This still has to be fixed, but can I do it a separate PR?

Reply:
What's wrong with it now? That there could be a `countries.sum_gross_cost`?

Reply:
That and the fact that these aggregation functions should not be here. Instead there should be columns referenced by name.

Reply:
Ah, sorry, I was myopically looking at just that line. Makes sense now